MAINTAINING.md

```diff
@@ -119,19 +119,30 @@ complete that no input from the original developer or maintainer is required.
 
 ### Organisation of reviewing between maintainers
 
-This section is a placeholder for further discussion around making the review
-process more efficient and effective when multiple maintainers are involved.
-This could include:
+#### Partial review
 
--   Usage of labels
--   Division of review responsibilities, e.g. with ACKs for different areas of
-    review
--   Usage of GitHub functionality, for example "Reviewers" and "Assignees" lists
+When you review a PR only partly according to the points outlined in the
+[technical guidelines](#technical-guidelines) please do not "approve" the PR to
+prevent accidental merges. Rather, give your verbal ACK and describe what you
+reviewed. In addition, if you processed or reasonably stepped over a whole
+section, mark the PR with the according label from the "Reviewed:" category.
+If you disagree with the assessment of a previous review, you may remove a
+certain "Reviewed:" label.
+
+When all "Reviewed:" labels are set you may give your approval for the PR.
 
 #### Github etiquette
 
--   If there are multiple maintainers reviewing a PR, always give the other
-    maintainers reasonable time to ACK before dismissing their review.
+It is good etiquette to describe what you reviewed, even if you gave the PR a
+full review and gave your approval. This way the contributor and other
+maintainers are able to follow your thought process.
+
+Only assign yourself to PRs. You can however request reviews from other
```

Comment:
This seems completely new to me. What it discussed during the summit? I don't complain about it, just saying that we might want to communicate this 'new' rule better.

Reply:
Hi yes it was discussed during the summit.

> I don't complain about it, just saying that we might want to communicate this 'new' rule better.

That is what this PR is for. To communicate and discuss the rules introduced here.

Reply:
As a maintainer, if I arrive on a PR done by A where B is assigned, I assumes that B takes care of it. Same with an issue. And so that B tries to dedicate some of his time to work on the PR/issue.

When requesting "review", I understand it more nicely with "if you can look at it if you have time it would be great".